src/comparators.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Sort comparators for selection-based components.
+ */
+const sortComparators = {
```

Comment:
Why have them in this object? Why not export each one?

`export const ascByLabel = (a, b) => {...};`